awx_collection/tools/vars/resolution.yml

```diff
@@ -0,0 +1,6 @@
+---
+name_to_id_endpoint_resolution:
+  webhook_credential: credentials
```

Comment:
So this is done strictly for ForeignKey objects?

Reply:
Yup, because an association can have a "weird" name.